backend/src/api/player_view_functions.clj

```diff
@@ -2,31 +2,31 @@
   (:require [rules.victory-conditions :as victory]
             [configs.messages :as messages]))
 
+(defn ^:private partial-card
+  "Returns only certain keys of a card and change the :owner"
```

Comment:
I know that this could be a more generic function (no need to call it "card") but since it changes the `:owner`, I felt like it should be like that.

Also, considering it's private, I see no reason to make it more generic.

Reply:
agree with no need to be more generic